platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/view/MapViewPager.java

```diff
@@ -13,8 +13,8 @@ public MapViewPager(Context context, AttributeSet attrs) {
     super(context, attrs);
   }
 
-  @Override
-  protected boolean canScroll(View v, boolean checkV, int dx, int x, int y) {
-    return v instanceof SurfaceView || v instanceof PagerTabStrip || (super.canScroll(v, checkV, dx, x, y));
-  }
+//  @Override
```

Comment:
Good to remove below lines?

Reply:
My bad, left over from testing, these lines should remain